webapp/channels/src/components/channel_settings_modal/channel_settings_access_rules_tab.test.tsx

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {shallow} from 'enzyme';
```

Comment:
Can we please not use enzyme? We're trying to actively move away from enzyme and use React Testing Library to allow us to, among other things, update react version.